docs/scriptappy.json

```diff
@@ -1973,6 +2017,25 @@
         }
       }
     },
+    "ContainerComponent": {
```

Comment:
You forgot to build the spec

Reply:
done
(many changes in the spec, probably not any rebuild since the last rebase from master)